plugins/legacy-plugin-chart-pivot-table/src/PivotTable.js

```diff
@@ -28,10 +29,7 @@ import { formatCellValue, formatDateCellValue } from './utils/formatCells';
 import fixTableHeight from './utils/fixTableHeight';
 import 'datatables.net-bs/css/dataTables.bootstrap.css';
 
-if (window.$) {
-  dt(window, window.$);
-}
-const $ = window.$ || dt.$;
+dt(window, $);
```

Comment:
Does this break since `$` will now be undefined.

Reply:
see line 22